core/new-gui/src/app/dashboard/component/top-bar/user-icon/user-icon.component.html

```diff
@@ -1,19 +1,20 @@
-<div ngbDropdown class="d-inline-block user-icon">
-  <button *ngIf="!user?.name;else user_avatar" class="btn btn-outline-primary" id="dropdownBasic1" ngbDropdownToggle>
-    Sign In
-  </button>
-  <ng-template #user_avatar>
+
+<div class="user-icon">
+  <button *ngIf="!user" nz-button nzType="default" nzSize="large" (click)="onClickLogin()">Sign In</button>
```

Comment:
change this to default makes the button not very visible in Workspace. Do we want to change the css to make it more noticable?

Reply:
changed, now it looks very similar to what we currently have